src/app/garfile/responsibleperson/index.njk

```diff
@@ -26,147 +26,149 @@
-              <label class="govuk-label" for="responsibleAddressLine1">{{__('field_house_number')}}<span class="govuk-visually-hidden">{{__('field_line_1_of_2')}}</span></label>
+              <label class="govuk-label" for="responsibleAddressLine1">{{ __('field_house_number') }}<span class="govuk-visually-hidden">{{ __('field_line_1_of_2') }}</span></label>
               {{ m.error_message('responsibleAddressLine1', errors, __) }}
               <input class="govuk-input"
                 id="responsibleAddressLine1" name="responsibleAddressLine1" type="text"
-                value="{{cookie.getGarResponsiblePerson().responsibleAddressLine1}}"
+                value="{{ cookie.getGarResponsiblePerson().responsibleAddressLine1 }}"
                 maxlength="{{ MAX_STRING_LENGTH }}"
                 {{ 'aria-describedby=responsibleAddressLine1-error' if errors | containsError('responsibleAddressLine1') }}>
             </div>
 
             <div class="govuk-form-group {{ m.form_group_class('responsibleAddressLine2', errors) }}">
-              <label class="govuk-label" for="responsibleAddressLine2"><span class="govuk-visually-hidden">{{__('field_line_2_of_2')}}</span></label>
+              <label class="govuk-label" for="responsibleAddressLine2"><span class="govuk-visually-hidden">{{ __('field_line_2_of_2') }}</span></label>
               {{ m.error_message('responsibleAddressLine2', errors, __) }}
               <input class="govuk-input"
                 id="responsibleAddressLine2" name="responsibleAddressLine2" type="text"
-                value="{{cookie.getGarResponsiblePerson().responsibleAddressLine2}}"
+                value="{{ cookie.getGarResponsiblePerson().responsibleAddressLine2 }}"
                 maxlength="{{ MAX_STRING_LENGTH }}"
                 {{ 'aria-describedby=responsibleAddressLine2-error' if errors | containsError('responsibleAddressLine2') }}>
             </div>
 
             <div class="govuk-form-group {{ m.form_group_class('responsibleTown', errors) }}">
-              <label class="govuk-label" for="responsibleTown">{{__('field_town')}}</label>
+              <label class="govuk-label" for="responsibleTown">{{ __('field_town') }}</label>
               {{ m.error_message('responsibleTown', errors, __) }}
               <input class="govuk-input"
                 id="responsibleTown" name="responsibleTown" type="text"
-                value="{{cookie.getGarResponsiblePerson().responsibleTown}}"
+                value="{{ cookie.getGarResponsiblePerson().responsibleTown }}"
                 maxlength="{{ MAX_STRING_LENGTH }}"
                 {{ 'aria-describedby=responsibleTown-error' if errors | containsError('responsibleTown') }}>
             </div>
 
+            <div class="govuk-form-group {{ m.form_group_class('responsiblePostcode', errors) }}">
+              <label class="govuk-label" for="responsiblePostcode">{{ __('field_postcode') }}</label>
```

Comment:
Question: Was there any particular reason the the postcode field was reordered?

Reply:
The FBO Answer section and Address section were in one fieldset.  So, I improved the form by putting them in separate fieldsets.

I thought the address section could be improved by putting the postcode field after the Town/City field.  This seems to be the more intuitive design.  Do you agree?

Previous Address format:
<img width="652" alt="4a Address Old Format" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/132352825/38a3ed78-7135-4f45-977e-fb6493a45aac">

Updated Address format:
<img width="656" alt="4b Address New Format" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/132352825/27f50c2b-7663-4b43-abfa-f4c20248ede2">

Reply:
If I am honest, I can't see one of the design looking much better than another.
* However, I am ok for the design change to be added.